packages/tupaia-web/src/features/Map/MapOverlaySelector/MapOverlaySelector.tsx

```diff
@@ -6,12 +6,16 @@
 import React, { useState } from 'react';
 import { DesktopMapOverlaySelector } from './DesktopMapOverlaySelector';
 import { MobileMapOverlaySelector } from './MobileMapOverlaySelector';
+import { gaEvent } from '../../../utils';
 
 export const MapOverlaySelector = () => {
   const [overlayLibraryOpen, setOverlayLibraryOpen] = useState(false);
 
   const toggleOverlayLibrary = () => {
     setOverlayLibraryOpen(!overlayLibraryOpen);
+    if (overlayLibraryOpen === false) {
+      gaEvent('Dropdown', 'Close');
```

Comment:
This should be called with `Overlay` `Close` if it's meant to be the same as the old one

Reply:
I've removed this one altogether. I got a bit confused with some of the naming conventions in the old one and thought dropdown overlays were called every time a dropdown menu was opened. Tom explained it to me and we don't think the dropdown menus need them at all.